src/netaddr.c

```diff
@@ -590,7 +590,7 @@ static pr_netaddr_t *get_addr_by_ip(pool *p, const char *name,
 static pr_netaddr_t *get_addr_by_name(pool *p, const char *name,
     array_header **addrs) {
   pr_netaddr_t *na = NULL;
-  int res;
+  int res, save_errno;
```

Comment:
Style nit:  I tend to use a variable name of `xerrno` for just this purpose; you'll see lots of `xerrno` variables in the ProFTPD codebase.